ecommerce/docs/STEP15-16_PLAN.md

```
@@ -0,0 +1,412 @@
+# STEP 15-16 과제 계획서
+
+> **작성일**: 2025-12-11
+> **목표**: Application Event 기반 관심사 분리 + 분산 트랜잭션 설계
+
+---
+
+## 빠른 실행 가이드
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)